go/runtime/host/sgx/sgx.go

```diff
@@ -421,6 +438,12 @@ func New(cfg Config) (host.Provisioner, error) {
 		cfg.RuntimeAttestInterval = defaultRuntimeAttestInterval
 	}
 
+	if metrics.Enabled() {
```

Comment:
We could also move this to `metrics.go` (e.g. create a new function `initMetrics`).